src/components/grid/GridTitelInfo.vue

```diff
@@ -15,50 +25,19 @@ import Vue from 'vue'
 export default Vue.extend({
   name: 'GridTitelInfo',
   props: {
-    info: {
-      type: Object,
+    name: {
+      type: String,
       required: true
+    },
+    label: {
+      type: String,
+      required: false
     }
   },
   computed: {
     variableName () {
-      return variable => variable.label ? variable.label : variable.name
+      return this.label ? this.label : this.name
     }
   }
 })
 </script>
-
-<style lang="scss" scoped>
-  @import "../../scss/variables";
-
-  .detail-info-hover {
-    position: relative;
-
-    .detail-info {
-      background-color: $light;
-      display: block;
-      opacity: 0;
-      padding-left: 1rem;
-      padding-right: 1rem;
-      pointer-events: none;
-      position: absolute;
-      z-index: 1;
-
-      path {
-        fill: $secondary;
-      }
-    }
-
-    .variable-title {
-      display: block;
-      overflow: hidden;
-      padding-left: 1rem;
-      padding-right: 1rem;
-      text-overflow: ellipsis;
```

Comment:
Some overflow handeling should remain. Variables names can now flow over table.

Reply:
Whoops, bit over-eager there.

Reply:
Reinstated the styling for the overflow.